### PR TITLE
Make AWS region configurable in the operator config map

### LINKED ISSUE #316: Missing configurable AWS zones. Currently hardcoded to "eu-central-1".
This feature is marked as _todo_, but just adding a +1 to that :)

https://github.com/zalando-incubator/postgres-operator/blob/master/pkg/util/constants/aws.go#L8

I think I'll have to fork & build, hardcoding my own zone as _the quickest possible fix_

Thank you for all the hard work on postgres-operator!

### PR DESCRIPTION
**Pull request by zerg-junior**
Closes #316

### CODE REVIEW

File: docs/reference/operator_parameters.md [hunk 1]
```diff
@@ -212,6 +212,9 @@ words.
   pods. Only used when combined with
   [kube2iam](https://github.com/jtblin/kube2iam) project on AWS. The default is empty.
 
+* **aws_region**
```
Comment (alexeyklyukin) [inline docs/reference/operator_parameters.md]: this should be in the "AWS or GSC interaction" section of the docs.

Comment (zerg-junior) [reply docs/reference/operator_parameters.md]: it already is :)

Comment (alexeyklyukin) [reply docs/reference/operator_parameters.md]: Missed that, sorry.

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**